emf-builtins/.project

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this an Eclipse specific file? Do we need this file to be committed? If so, could we add a line in the README of this project about this file?

Reply:
I'll add something to readme.